cmd/clusterctl/Dockerfile

```diff
@@ -28,5 +31,18 @@ RUN apk update && apk add ca-certificates openssh-client && rm -rf /var/cache/ap
 
 WORKDIR /root/
 
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates curl gnupg2 software-properties-common \
+    && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
+    && apt-get update && apt-get install -y docker-ce-cli
+
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl
```

Comment:
we can use `RUN curl -L -o /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl` so that you did not need to move the binary to `/bin` any more